test_cases/wof_localadmins.json

```diff
@@ -10,16 +10,17 @@
       "in": {
         "text": "Villares De Órbigo, Spain",
         "sources": "wof",
+        "lang": "es",
```

Comment:
Adding a language param here feels like changing the test rather than updating it to reflect new data, right? (although the results in Spanish _are_ quite nice :) )

Reply:
I'd prefer to not change the nature of these tests.  They're only testing particular layers of WOF data, not that they change language as there are already tests for that.